### PR TITLE
ghost role ruins are guaranteed spawns, ghost ruins no longer cost any points to spawn

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
ghost role ruins are guaranteed spawns, ghost ruins no longer cost any points to spawn

If you have any general comments on ghost ruins, throw them in this PR

## Why It's Good For The Game
I don't really think we should be locking behind joinable roles behind rng. I don't even really know what exactly I need to defend here because having a role that someone wants to play in a round followed by just not having that role available because of rnd rather than someone taking that role is not a very good idea. It also is the worst bandaid fix I've seen in a long time to make shittier roles seem like they're better. Rare != good.

The other reason I made this pr is to gather people's opinion on general ghost roles and which ones they find to be lacking/what other ghost roles succeed (For example i'm probably gonna add a stopwatch item to theta to let people time themselves like they're in a speedrun cuz that sounds fun and fits with the ruin)

The only semi hard sell in this is ash walkers, and although I find the ruin to not really be that interesting (It's pure combat and might as well have "no rp allowed" written on it) but i dont find ash walkers to actually be that much of a pest with them normally getting slaughtered.

## Testing
I tested this like 20 days ago, something probably changed and i'll look at the diff later but it worked fine testing it today again

## Changelog
:cl:
tweak: Ghost ruins now spawn 100% of the time, ghost ruins no longer take up slots for other ruins
/:cl:

### CODE REVIEW

**Comment by Sadhorizon** [comment]
Would every ghostrole spawn at once?
That sounds like it would take some real estate for other ruins, on lavaland especially.

Idea sounds interesting however.

> The other reason I made this pr is to gather people's opinion on general ghost roles and which ones they find to be lacking/what other ghost roles succeed

As for ghost roles. I can already say, lavaland doctor is a joke and is used only as a respawn point. You get to station and you basically are hired already, due to the loadout they are using. Only cool thing about them is their shotgun.

Other lacking ghostrole is beach gang. Their main issue is that crew won't really visit your bar, while miners are too busy gaming to bother.

And last ghost role that is lacking (although not as significantly) would be hermit. The whole gimmick of trying to survive is interesting, but could use an improvement.

Dionae are alright, they do what they were designed for quite well.
Theta is certainly fun. Personally, I don't think it needs any changes but maybe.

All of others also are quite alright, but I don't have much comments there.

**Comment by Exavere** [comment]
I don't believe having so many ruins spawning at once is all that great of a thing.  

Maybe guaranteeing a few select ruins wouldn't hurt, though that'd be a different thing like the conversation that'd have to be held over whether or not we want something like Ash Walkers roaming Lavaland every single round, but having this many roles up would just be very cramped.

**Comment by GDNgit** [comment]
> I don't believe having so many ruins spawning at once is all that great of a thing.  
> 
> Maybe guaranteeing a few select ruins wouldn't hurt, though that'd be a different thing like the conversation that'd have to be held over whether or not we want something like Ash Walkers roaming Lavaland every single round, but having this many roles up would just be very cramped.  

I don't think it would be very cramped at all. In total this adds 7 guaranteed ruins, for space this will change pretty much nothing. For lavaland this will just definitely make the z level a bit more crowded, but that's going to amount to like an extra biodome and sliding puzzle.
In the case with ash walkers, yes, ash walkers have issues. I don't think the way those issues should be delt with is through just making it random vs actually giving them something to do beyond just murdering people

**Comment by Exavere** [comment]
Being something for Miners to interact with + being hardmode lavaland survival is what they got goin' for them atm. 
I also am not aware of any ash walker issues being handled by just RNG, from what I know they're just meant to be a random thing that happens every now and then on the Lavaland which they do just fine. 

It mostly boils down to whether or not we want the presence of the Murderbone Antag on Lavaland every single round.  While I personally enjoy Ash Lizards killing Miners for funny haha I am at least aware that them being able to be a threat every single round may be a bit much.  Also, I'd rather they continue to be simple murderous threats.  Simple to play, play against, don't have to worry about being bwoinked for messing up your nest/tendril obligations; no need for another policy/semi-antag restriction fiasco with loads of grey area.  

That said, I cannot speak for the other design team members but I do believe they resonate similar opinions.

**Comment by Exavere** [comment]
I don't see any issue with it now.

**Review by Henri215 — APPROVED** [review]
Lewcs Good To Me

**Review by lewcc — APPROVED** [review]
Code looks good

File: code/datums/ruins/lavaland.dm [hunk 1]
```diff
@@ -13,6 +13,8 @@
 	description = "Seemingly plucked from a tropical destination, this beach is calm and cool, with the salty waves roaring softly in the background. \
 	Comes with a rustic wooden bar and suicidal bartender."
 	suffix = "lavaland_biodome_beach.dmm"
+	cost = 0
+	always_place = TRUE
```
Comment (Contrabang) [inline code/datums/ruins/lavaland.dm:17]: Beach biodome has been removed from the code I believe. Update this when you can please

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/datums/ruins/lavaland.dm (hunk 1 above)
Comment (lewcc) [reply code/datums/ruins/lavaland.dm:17]: Oof, yes. Please merge master if you haven't recently; good catch contra

Comment (GDNgit) [reply code/datums/ruins/lavaland.dm:17]: > Beach biodome has been removed from the code I believe. Update this when you can please

it's TMed back in rn, also yeah, the conflicted pr will need a merge master

**Review by Contrabang — APPROVED** [review]
LGTM

**Pull request merged**